src/js/node/http2.ts

```diff
@@ -1,10 +1,113 @@
 // Hardcoded module "node:http2"
+
+const { isTypedArray } = require("node:util/types");
+
 // This is a stub! None of this is actually implemented yet.
 const { hideFromStack, throwNotImplemented } = require("$shared");
 
-function connect() {
-  throwNotImplemented("node:http2 connect", 887);
-}
+const tls = require("node:tls");
+const net = require("node:net");
+const bunTLSConnectOptions = Symbol.for("::buntlsconnectoptions::");
+type Http2ConnectOptions = { settings?: Settings; protocol?: "https:" | "http:"; createConnection?: Function };
+const TLSSocket = tls.TLSSocket;
+const EventEmitter = require("node:events");
+const { Duplex } = require("node:stream");
+const primordials = require("internal/primordials");
+
+const { H2FrameParser, getPackedSettings, getUnpackedSettings } = $lazy("internal/http2");
+
+const sensitiveHeaders = Symbol.for("nodejs.http2.sensitiveHeaders");
+const bunHTTP2Native = Symbol.for("::bunhttp2native::");
+const bunHTTP2StreamResponded = Symbol.for("::bunhttp2hasResponded::");
+const bunHTTP2StreamReadQueue = Symbol.for("::bunhttp2ReadQueue::");
+const bunHTTP2Closed = Symbol.for("::bunhttp2closed::");
+const bunHTTP2Socket = Symbol.for("::bunhttp2socket::");
+const bunHTTP2WantTrailers = Symbol.for("::bunhttp2WantTrailers::");
+const bunHTTP2Session = Symbol.for("::bunhttp2session::");
+
+const ReflectGetPrototypeOf = Reflect.getPrototypeOf;
+const FunctionPrototypeBind = primordials.FunctionPrototypeBind;
+const isArray = Array.isArray;
```

Comment:
You can use `$isJSArray` instead of `Array.isArray`.

Reply:
Or @paperdave maybe we could add Array.isArray to the list of replacements?

Reply:
fixed in https://github.com/oven-sh/bun/pull/6761/commits/946a993adea0d7084e96012fd60337da5be5bc49